Bio/Phylo/Applications/_Phyml.py

```diff
@@ -33,39 +33,40 @@ class PhymlCommandline(AbstractCommandline):
     def __init__(self, cmd="phyml", **kwargs):
         """Initialize the class."""
         self.parameters = [
-            _Option(["-i", "--input", "input"],
-                    "PHYLIP format input nucleotide or amino-acid sequence filenam.",
-                    filename=True,
-                    is_required=True,
-                    equate=False,
-                    ),
-
-            _Option(["-d", "--datatype", "datatype"],
-                    "Datatype 'nt' for nucleotide (default) or 'aa' for amino-acids.",
-                    checker_function=lambda x: x in ("nt", "aa"),
-                    equate=False,
-                    ),
-
-            _Switch(["-q", "--sequential", "sequential"],
-                    "Changes interleaved format (default) to sequential format."
-                    ),
-
-            _Option(["-n", "--multiple", "multiple"],
-                    "Number of data sets to analyse (integer).",
-                    checker_function=(lambda x: isinstance(x, int) or x.isdigit()),
-                    equate=False,
-                    ),
-
-            _Switch(["-p", "--pars", "pars"],
-                    """Use a minimum parsimony starting tree.
+            _Option(
+                ["-i", "--input", "input"],
+                "PHYLIP format input nucleotide or amino-acid sequence filenam.",
+                filename=True,
+                is_required=True,
+                equate=False,
+            ),
+            _Option(
+                ["-d", "--datatype", "datatype"],
+                "Datatype 'nt' for nucleotide (default) or 'aa' for amino-acids.",
+                checker_function=lambda x: x in ("nt", "aa"),
+                equate=False,
+            ),
+            _Switch(
+                ["-q", "--sequential", "sequential"],
+                "Changes interleaved format (default) to sequential format.",
+            ),
+            _Option(
+                ["-n", "--multiple", "multiple"],
+                "Number of data sets to analyse (integer).",
+                checker_function=(lambda x: isinstance(x, int) or x.isdigit()),
+                equate=False,
+            ),
+            _Switch(
+                ["-p", "--pars", "pars"],
+                """Use a minimum parsimony starting tree.
 
                     This option is taken into account when the '-u' option is absent
                     and when tree topology modifications are to be done.
-                    """
-                    ),
-
-            _Option(["-b", "--bootstrap", "bootstrap"],
-                    r"""Number of bootstrap replicates, if value is > 0.
+                    """,
```

Comment:
This looks wrong - the opening and closing triple quotes are not aligned, and changing the opening line only probably changes the interpretation of the indentation of the text?

Reply:
Yes, you are right, looks odd, I'll align them, it just needs to be done manually.

Reply:
I was wrong about how Python interprets these strings - but these help text strings were and still are not optimal. See #2369 and main comment.